plugins/lua/buildingplan.lua

```diff
@@ -0,0 +1,3 @@
+local _ENV = mkmodule('plugins.buildingplan')
+
```

Comment:
Lua API docs would be great - simple is fine for these functions, just as long as people know that the functions exist. Listing them in `Lua API.rst` is probably more important than listing them here, but seeing this reminded me. (If you wanted to wait for a  confirmation before documenting these functions, they look sensible to me.)

Reply:
done. thanks!